.github/workflows/coverage.yaml

```diff
@@ -0,0 +1,29 @@
+on: [push]
+
+name: build
+
+env:
+  RUST_BACKTRACE: 1
```

Comment:
Is this intended to land or just for debugging this PR? Is this needed for coverage or is this for debugging when things do go wrong?

Reply:
Both, it doesn't hurt to have it, it's not needed but it's better to have it enabled in CI

Reply:
Sounds good!